cli/ts/commands/verify.ts

```diff
@@ -111,7 +113,9 @@ export const verify = async ({
 
   const pollContract = new BaseContract(pollAddr, parseArtifact("Poll")[0], ethSigner) as Poll;
 
-  const tallyContract = new BaseContract(tallyContractAddress, parseArtifact("Tally")[0], ethSigner) as Tally;
+  const tallyContract = useQv
+    ? (new BaseContract(tallyContractAddress, parseArtifact("Tally")[0], ethSigner) as Tally)
+    : (new BaseContract(tallyContractAddress, parseArtifact("TallyNonQv")[0], ethSigner) as TallyNonQv);
```

Comment:
Is it possible to have one interface here?

Reply:
I'm afraid not, unless we want to have unused parameters in certain functions